app/controllers/lists_controller.rb

```diff
@@ -95,6 +105,23 @@ def update
   #   redirect_to lists_url, notice: 'List was successfully destroyed.'
   # end
 
+  # GET /lists/:id/associations/entities
+  def new_entity_associations; end
+
+  # POST /lists/:id/associations/entities
+  # only handles json
+  def create_entity_associations
```

Comment:
+1. so to do that, would we just emulate the pre-action hooks in the relatoinships controller?

Reply:
yup, all it needs is something something like this:
``` rb
before_action -> { check_permission('bulker') }
```

Reply:
instead of this, created `ApplicationController#block_unless_bulker` for use here, in the bulk relationship tool, and anything else that might need to check for bulk permissions. (see subsequent commit.)